cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const customerLoginBtn = 'Customer Login';
+  const user = 'Hermoine Granger';
+  const depositAmount = '100';
+  const withdrawAmount = '250';
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione_s bank account', () => {
+    cy.contains('.btn', customerLoginBtn).click();
+    cy.url().should('contain', 'customer');
+    cy.get('#userSelect').select(user);
+    cy.get('[type="submit"]').click();
+
+    cy.get('.fontBig').should('contain', user);
+    cy.get('#accountSelect').should('contain.text', '1001');
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '0')
+      .should('be.visible');
+
+
+    cy.contains('.ng-binding', 'Dollar').should('be.visible');
+    cy.get('[ng-click="deposit()"]').click();
+    cy.findByPlaceholder('amount').should('be.visible').type(depositAmount);
+    cy.contains('[type="submit"]', 'Deposit').click();
+    cy.get('[ng-show="message"]').should('contain', 'Deposit Successful');
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', depositAmount)
```

Comment:
check on ui mode for this code. it doesn't check the expected balance.
```suggestion
      .contains('strong', depositAmount)
```